cli/jobs/automl-standalone-jobs/cli-automl-image-classification-multiclass-task-fridge-items/cli-automl-image-classification-multiclass-task-fridge-items-automode.yml

```diff
@@ -0,0 +1,25 @@
+$schema: https://azuremlsdk2.blob.core.windows.net/preview/0.0.1/autoMLJob.schema.json
```

Comment:
Please add GitHub actions to run these yaml files, similar to non-automode yaml files.

Reply:
Will do, thanks for reminder.